.env

```diff
@@ -0,0 +1,2 @@
+DB_HOST=localhost
+DB_USER=jerrylinew
```

Comment:
If possible could we add a comment here to make sure it gets to peoples attention that they need to change this?